dsl-interpreter/src/client.ts

```diff
@@ -0,0 +1,45 @@
+import { Connection, WorkflowClient } from '@temporalio/client';
+import { DSLInterpreter, DSL } from './workflows';
+// @ts-ignore
```

Comment:
Why is this ignored?

Reply:
node-yaml doesnt have types, and our TS setup didnt recognize the .d.ts that i put in there and i couldnt figure it out. actually would like some help here.